templates/slotViz.html

```diff
@@ -416,40 +403,46 @@
         if (!slotVizCountdown) {
           slotVizCountdown = document.createElement("div")
           slotVizCountdown.className = "mb-3"
-          $("#" + selector).prepend(slotVizCountdown)
+          $(selector).prepend(slotVizCountdown)
         }
         const slots = Math.ceil((hfTs - now) / 1000 / secondsPerSlot)
-        slotVizCountdown.innerHTML = `<b>Hardfork in:</b> ${getRelativeTime(luxon.DateTime.fromMillis(hfTs))} | ${slots} slots`
+        slotVizCountdown.innerHTML = `<b>${hardforkName} Hardfork in:</b> ${getRelativeTime(luxon.DateTime.fromMillis(hfTs))} | ${slots} slots`
```

Comment:
I have just noticed that the word "in" is shown twice:
![image](https://user-images.githubusercontent.com/114066135/225239280-dc2499d9-56cf-471d-836a-e4558480b62a.png)

This is caused by https://github.com/gobitfly/eth2-beaconchain-explorer/blob/c3c4adb6bbdf09329a6fc6c6e862d9a217b07188/static/js/layout.js#L497
and
https://github.com/gobitfly/eth2-beaconchain-explorer/blob/228994874181d60551b15675d1e8b8ca6a299efa/templates/slotViz.html#L409
both adding "in".

My suggestion is to remove it in slotViz.html as `getRelativeTime` is used in many different locations.
```suggestion
        slotVizCountdown.innerHTML = `<b>${hardforkName} Hardfork</b> ${getRelativeTime(luxon.DateTime.fromMillis(hfTs))} | ${slots} slots`
```
(notice that I would also remove the colon so that the info "flows" better)

Reply:
removed the in: